src/components/hero.tsx

```diff
@@ -0,0 +1,74 @@
+import { Link } from "gatsby"
+import React from "react"
+
+import Button from "./button"
+import Text from "./text"
+
+type Props = {
+  allEpisodes: any
+}
+
+export default function Hero({ allEpisodes }: Props) {
+  const textStyle =
+    "text-white bg-black items-end flex justify-end p-2 my-2 mx-auto"
+  const imgStyle =
+    "bg-cover bg-center h-full ease-in-out transition-all duration-300 hover:opacity-95 flex items-end hover:scale-95"
+  return (
+    <div className="h-screen my-auto -mb-32 md:-mb-48 w-3/4 m-auto">
+      <div className="flex-col md:flex md:flex-row gap-4 h-2/3">
+        <div className="flex-1 mb-4 md:mb-0 h-52 md:h-full">
+          <Link to={`/podcast/${allEpisodes[0].guid}`}>
+            <div
+              className={imgStyle}
+              style={{
+                backgroundImage: `url(${allEpisodes[0].itunes.image})`,
+              }}
+            >
+              <Text as="h3" className={textStyle}>
+                {allEpisodes[1].title}
+              </Text>
+            </div>
+          </Link>
+        </div>
+        <div className="flex-1 flex flex-col gap-4">
+          <div className="bg-cover bg-center h-52 md:h-1/2 ">
+            <Link to={`/podcast/${allEpisodes[1].guid}`}>
```

Comment:
Why haven't you created a component like

```
<HeroCard
  slug={}
  title={}
  backgroundImage={}
  size="xl" <---- could be sm or md or xl
/>
```

Use cva for the variant

Reply:
cva broke everythings
<img width="1293" alt="CleanShot 2023-09-14 at 00 51 05@2x" src="https://github.com/flexbox/artaufeminin/assets/81434852/99de4c6e-c78b-4f09-b599-cbb966c2c4ca">